### ISSUE TITLE
Test direct bibliography import to server with eXist 5.0

### ISSUE THREAD

**Issue by nathangibson** [biblia-arabica/srophe-eXist-app#110]
To the extent that previous bugs have been worked out in more recent versions of eXist, is it likely that we could start importing bibliography reliably using zotero2bibl on the live server if we upgrade it to eXist 5.0? Could you perhaps test this locally or whatever? 
Or do you think we should just accept on principle that this kind of live updating is never a good idea? 
Looking for a less complicated pathway than the current one ...

**Comment by wsalesky** [comment]
@nathangibson we can test it, but there is an issue with eXist 5.0 with the app. (Maybe a 2 hour fix)

**Comment by nathangibson** [comment]
I see. Is this a bug in eXist that will be fixed? Or is it an intentional change that we'll need to adapt to eventually anyway?

**Comment by wsalesky** [comment]
It is an intentional change so we would need to adapt if we want to move to the 5.0 release.

**Comment by nathangibson** [comment]
OK, this is probably more important than IIIF Viewer #107 but less important than the other things so please do it if you have time.

**Comment by wsalesky** [comment]
@nathangibson I will try to make time for it this week.

**Comment by wsalesky** [comment]
@nathangibson This works locally, and the change to get it running on 5.0 is quite small, less then an hour. I tried to get it up on AWS last night but ran out of time. I can try again tonight, or if you  have an instance of 5.0 (even a RC version should be fine) you could load the zoter2bibl module and test it out, see if you can connect to Zotero.

**Comment by nathangibson** [comment]
@wsalesky At the moment I don't really have a server testing instance for biblia arabica. Can you test on your AWS and then if you think it's safe we can work on upgrading the production server and testing it there (with appropriate backups)?

**Comment by wsalesky** [comment]
@nathangibson Yes I can. My AWS installation failed last week, but I will give it another try this week.

**Comment by nathangibson** [comment]
Thanks. If it's a lot of effort, you can wait until we get our test server up and running instead. It just may be a while.

**Comment by wsalesky** [comment]
It is good practice for me. I will get it up, but probably not this week.
